### PR TITLE
Release multivariate flags

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

**DO NOT MERGE!** We need to rebase after base PR is merged.

- Releases multivariate flags for everyone. This feature has been enabled on Cloud for several weeks and was already included in the last self-hosted release.

## How did you test this code?
- Opened the app without the flag and made sure the multivariate option showed up.

### CODE REVIEW

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx [hunk 1]
```diff
@@ -277,199 +274,192 @@ export function FeatureFlag(): JSX.Element {
                         </Col>
                     </Row>
 
-                    {enabledFeatureFlags[FEATURE_FLAGS.MULTIVARIATE_SUPPORT] && (
```
Comment (paolodamico) [inline frontend/src/scenes/feature-flags/FeatureFlag.tsx:280]: Only change from here to the bottom of the file was removing this check and indentation automatically adjusted.

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx [hunk 2]
```diff
@@ -277,199 +274,192 @@ export function FeatureFlag(): JSX.Element {
...
+                                    options={[
+                                        {
+                                            label: 'Boolean value (A/B test)',
+                                            value: false,
+                                        },
+                                        {
+                                            label: (
+                                                <div>
+                                                    String value (Multivariate test){' '}
+                                                    <LemonTag type="warning">Beta</LemonTag>
```
Comment (paolodamico) [inline frontend/src/scenes/feature-flags/FeatureFlag.tsx:302]: Only material change. We upgraded this to Beta and use the new lemon tags.

**Review by mariusandra — APPROVED** [review]

**Comment by neilkakkar** [comment]
cc: @kpthatsme ! Think this is alright, but [last I asked in the Growth slack ](https://posthog.slack.com/archives/C018RGRS10S/p1638878213096800), library support was a blocker for releasing these everywhere.

Also, q: Do we want to make this a paid feature?

**Comment by paolodamico** [comment]
I think it's fine to release this to everyone as it is. Already valuable for users and it's marked as beta, but it should definitely be paid. Given that we're already including a ton of commits on top of the release, will not include this feature in the release. I'll add a pay gate later in the week.

**Comment by kpthatsme** [comment]
> cc: @kpthatsme ! Think this is alright, but [last I asked in the Growth slack ](https://posthog.slack.com/archives/C018RGRS10S/p1638878213096800), library support was a blocker for releasing these everywhere.
> 

Right, yea - the main reasons are the library support and a more philosophical one of nobody was fully committed to owning and granularly looking at it for correctness, etc. prior to shipping. Now that the team is dedicated to it I don't think that's so much of a reason to wait.

**Comment by paolodamico** [comment]
Alright added a pay gate here for Scale and above.

**Comment by kpthatsme** [comment]
> Alright added a pay gate here for Scale and above.
> 
> 

Would be nice if we could add a link here for them to upgrade - probably something we should just standardize though so not critical for getting this out

**Pull request merged**